README.md

```diff
@@ -122,26 +122,33 @@ If your change is intended to be released under the `latest` tag on npm, you mus
+- Create a new branch with the `beta-*` prefix, and push this to the remote. E.g. `git push origin feature-myawesomework`.
+- Create another branch, off this initial beta branch, to implement your code changes. Ensure that this branch **does not** use a prefix.
+- When you create your PR, target the `beta-*` branch.
+- Upon merging to your `beta-*` branch, a new PR titled **release: Release Packages (beta)** is automatically created. This PR includes the `CHANGELOG.md` and `package.json` version bump. Merging this PR will execute a publish to npm using the appropriate tag.
+
+
 ### Feature versions - 'next' tag
 
 A Feature release is for larger changes that may require multiple PRs, across several packages, before it is released. These changes are unstable and are not intended to be used by consumers. Typically, these releases will be used for testing changes in consuming applications as an alternative to using something like `yalc`.
 
+If your change is intended to be released under the `next` tag on npm, you must follow this workflow:
+
+- Create a branch with your changes. These changes should exclude any `package.json` or manual `CHANGELOG` updates – only include the `.changesets` changes added by Changesets.
+- When you create your PR, target the `staging` branch.
+- Upon merging to `staging`, a new PR titled **release: Release Packages (next)** is automatically created. This PR includes the `CHANGELOG.md` and `package.json` version bump. Merging this PR will commit these changes to `staging` and execute a publish to npm under the `next` tag.
 
-If your change is intended to be released under the `next` / `beta` tag on npm, you must follow this workflow:
-- Create a new branch with the `feature-*` / `beta-*` prefix, and push this to the remote. E.g. `git push origin feature-myawesomework`.
-- Create another branch, off this initial feature/beta branch, to implement your code changes. Ensure that this branch **does not** use a prefix.
-- When you create your PR, target the `feature-*` / `beta-*` branch.
-- Upon merging to your `feature-*` / `beta-*` branch, a new PR titled **release: release Packages (beta)** / **release: release Packages (next)** is automatically created. This PR includes the `CHANGELOG.md` and `package.json` version bump. Merging this PR will execute a publish to npm using the appropriate tag.
 
-**Notes:** Any new PRs that target the `feature-*` / `beta-*` branch will cause GitHub actions to include the changes as part of that beta/feature release.
-Any package that uses the `beta` / `next` tag **must** follow this workflow until it's ready to be promoted to the `latest` tag (see Stable Versions section). PRs that combine changes in `latest` and `beta` / `next` packages will result in the beta / next package being versioned incorrectly.
+**Notes:** Any new PRs that target the `beta-*` / `staging` branch will cause GitHub actions to include the changes as part of that beta release.
```

Comment:
A small suggestion.

What I usually do is refer to a version number as a "release" and anything that you deploy as a "deployment".

Releases always have deployments. Deployments are not always for a release as they may not always have a version number attached to it (for example when you deploy a PR's feature branch).

```suggestion
**Notes:** Any new PRs that target the `beta-*` / `staging` branch will cause GitHub actions to include the changes as part of that beta deployment.
```

Reply:
Thanks for this! I just realised I need to change this to `beta / next deployment` :D

Reply:
Hehe I was a bit confused, but I didn't wanna comment while not fully in context of your development lifecycle / CI yet.